packages/interactive_media_ads/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
```

Comment:
`## NEXT`

(See comment in pubspec.yaml.)

Reply:
Spoke offline about the initial release